### ISSUE TITLE
wxTextCtrl problem.

### ISSUE THREAD

**Issue by Samo99** [wangqr/Aegisub#101]
Hey man, Thank you so much for wxTextCtrl fix.

But the punctuation still appear in the opposite direction if you type it at the end of a sentience like before even after disabling the wxSTC. And you need it type it first if you want it to be in the correct position.

ex:

![](https://user-images.githubusercontent.com/12115578/113537260-d85b3100-95e0-11eb-8a2c-228ff2fd5cda.png)

![](https://user-images.githubusercontent.com/12115578/113537263-d98c5e00-95e0-11eb-872e-6f6fef5217d1.png)

**Comment by wangqr** [comment]
Can you provide a screenshot of how this line looks like in video preview or in an actual media player?

**Comment by Samo99** [comment]
> Can you provide a screenshot of how this line looks like in video preview or in an actual media player?

OK

This is the preview for the first pic 
![1](https://user-images.githubusercontent.com/12115578/114226679-8ae21980-997c-11eb-98c6-aaf0d03fb16b.png)


preview for the second pic 

![2](https://user-images.githubusercontent.com/12115578/114226728-a0efda00-997c-11eb-97c3-c0a72da8ee86.png)

**Comment by wangqr** [comment]
As a subtitle editor, whichever layout that matches player output is considered correct. But unfortunately we don't have control over how Windows text box show those right to left text mixed with left to right text.
You may try toggling the "right to left reading order" option in the shift+right click menu and see if that helps.

**Comment by Samo99** [comment]
> As a subtitle editor, whichever layout that matches player output is considered correct. But unfortunately we don't have control over how Windows text box show those right to left text mixed with left to right text.
> You may try toggling the "right to left reading order" option in the shift+right click menu and see if that helps.

Yeah that's a problem in windows andi hoped that your workaround fix it like Microsoft WORD, cause it's the only app with proper solution for this problem.

I look at your suggestion and it's already enabled but i tried to go in same menu to insert Unicode control character and from that i clicked on RTE and it fixed it, but you need to repeat these steps on every line.
If there is a way to enable Unicode RTE for every line automatically in Aegisub that will be great.

**Comment by wangqr** [comment]
If you want to prepend something to every line, you can use `Edit -> Find and replace`, enable "Use regular expressions", and replace `^` with the prefix you need. You may also make a lua script and bind it to some hotkey if it is used frequently.

**Comment by Samo99** [comment]
> If you want to prepend something to every line, you can use `Edit -> Find and replace`, enable "Use regular expressions", and replace `^` with the prefix you need. You may also make a lua script and bind it to some hotkey if it is used frequently.

Using find and replace method is a solution but if it can be an option inside Aegisub that i can enable or disable would be great.
And about o a script, hehe! i which i knew how to program :)
